src/block_cache/block_chunkcache.c

```diff
@@ -20,6 +20,43 @@
 #define WT_CHUNK_OFFSET(chunkcache, offset) \
     (wt_off_t)(((size_t)offset / (chunkcache)->chunk_size) * (chunkcache)->chunk_size)
 
+/*
+ * __bitmap_find_free --
+ *     Iterate through the bitmap to find a free chunk in the cache.
+ */
+static int
+__bitmap_find_free(WT_SESSION_IMPL *session, size_t *bit_index)
+{
+    WT_CHUNKCACHE *chunkcache;
+    size_t bitmap_size, bits_remainder;
+    uint8_t map_byte;
+
+    chunkcache = &S2C(session)->chunkcache;
+    bitmap_size = (chunkcache->capacity / chunkcache->chunk_size) / 8;
+
+    /* Iterate through the bytes and bits of the bitmap to find free chunks */
+    for (size_t i = 0; i < (bitmap_size); i++) {
```

Comment:
In WT I have mostly seen the variable declared at the top of the functions (even for 'for loops')